tusclient/uploader.py

```diff
@@ -102,8 +125,7 @@ def get_offset(self):
         resp = requests.head(self.url, headers=self.headers)
         offset = resp.headers.get('upload-offset')
         if offset is None:
-            msg = 'Attemp to retrieve offset fails with status {}'.format(resp.status_code)
-            raise TusCommunicationError(msg, resp.status_code, resp.content)
+            raise TusCommunicationError('', resp.status_code, resp.content)
```

Comment:
It's always preferable to have error messages as precise as possible, and that have saved my countless hours, especially when debugging tus client issues :) In particular, details on what request method and url failed and what headers are missing/malformed, if any, are useful.